lib/bolt/node/ssh.rb

```diff
@@ -19,8 +20,10 @@ def protocol
     end
 
     def connect
+      transport_logger = Logging.logger[Net::SSH]
+      transport_logger.level = :warn
       options = {
-        logger: @transport_logger,
```

Comment:
What was transport_logger used for?

Reply:
It's a separate logger that gets passed to `Net::SSH`, so that we can control the verbosity of its messages separate from the rest of the system.

Reply:
Ah, but you got rid of the `@transport_logger` variable.